charts/argo-cd/Chart.yaml

```diff
@@ -4,7 +4,7 @@ version: 0.0.20
 description: A Helm chart for Argo CD, a declarative, GitOps continuous delivery tool for Kubernetes.
```

Comment:
😭 the version is too far away :)

Reply:
wdym?

Reply:
I cannot create the commit to change it from the PR view.